name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/AbstractIndicatorWidget.java

```diff
@@ -75,6 +107,28 @@ public Control getTitleControl()
     @Override
     public void update(D data)
     {
+        // construct label to indicate the data series (user can manually change
+        // the label later)
+        getWidget().setLabel(WidgetFactory.valueOf(getWidget().getType()).getLabel() + ", " //$NON-NLS-1$
```

Comment:
This code will prevent that the user can edit the label b/c you override the edited version with the default title.